lib/stream.js

```diff
@@ -255,3 +256,7 @@ module.exports = class UDXStream extends streamx.Duplex {
 }
 
 function noop () {}
+
+function toBuffer (data) {
+  return b4a.isBuffer(data) ? data : b4a.from(data)
```

Comment:
i think typeof data === string ? b4a.from(data) : data is a bit safer as it's more conservative (would prefer an array crashing the process with an assertion later on)

Reply:
can you b4a the other allocs in here, might as well